.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -151,13 +149,14 @@ After you have the version increments, the updated checkpoints and the updated c
 - [ ] Test that the newly built Zebra starts correctly, by running `~/.cargo/bin/zebrad`
 - [ ] Wait until the [Docker binaries have been built on `main`](https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml), and the quick tests have passed.
       (You can ignore the full sync and `lightwalletd` tests, because they take about a day to run.)
-- [ ] Publish the release to GitHub by disabling 'pre-release', then clicking "Publish Release"
+- [ ] [Publish the release to GitHub](https://github.com/ZcashFoundation/zebra/releases) by disabling 'pre-release', then clicking "Publish Release"
 - [ ] Wait until [the Docker images have been published](https://github.com/ZcashFoundation/zebra/actions/workflows/release-binaries.yml)
+- [ ] Test the Docker image using `docker run zfnd/zebra`
```

Comment:
This needs some bug fixes at the moment, but it should work before the next release:
- #5312 
- #5313

Reply:
```suggestion
- [ ] Test the Docker image using `docker run zfnd/zebra:1.0.0-rc.<version>`
```